src/Atoms/Exercise/Exercise.js

```diff
@@ -1,13 +1,13 @@
-import propTypes from 'prop-types';
+import CLASSES from '../../Data/Dummy-data/test-data-classes';
 import ExerciseStyled from './ExerciseStyled';
 import PropTypes from 'prop-types';
```

Comment:
Is there a reason for propTypes and PropTypes to be imported ?

Reply:
Out of scope.